## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 440426`
Commit: `731f190b1d60a96a247e6d49e89f99670be18320`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>plakar</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>plakar</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>plakar</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>plakar</summary>
<pre>Building subPackage ./utils/btreescan
Building subPackage ./versioning
buildPhase completed in 2 minutes 2 seconds
Running phase: checkPhase
ok      github.com/PlakarKorp/plakar/api        0.059s
ok      github.com/PlakarKorp/plakar/appcontext 0.024s
ok      github.com/PlakarKorp/plakar/btree      0.008s
ok      github.com/PlakarKorp/plakar/chunking   0.005s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/agent       0.448s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/archive     0.131s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/backup      0.235s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/cat 0.760s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/check       0.302s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/clone       0.170s
ok      github.com/PlakarKorp/plakar/cmd/plakar/subcommands/config      0.024s
# github.com/PlakarKorp/plakar/cmd/plakar/subcommands/create [github.com/PlakarKorp/plakar/cmd/plakar/subcommands/create.test]
cmd/plakar/subcommands/create/create_test.go:120:16: undefined: syscall.Dup2
cmd/plakar/subcommands/create/create_test.go:122:16: undefined: syscall.Dup2
FAIL    github.com/PlakarKorp/plakar/cmd/plakar/subcommands/create [build failed]
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/pl/plakar/package.nix
+++ b/pkgs/by-name/pl/plakar/package.nix
@@ -1,0 +1,48 @@
+{
+  stdenv,
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fuse,
+}:
+buildGoModule (finalAttrs: {
+  pname = "plakar";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "PlakarKorp";
+    repo = "plakar";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-4fJB00poA06aQC//zw6L0bRXa8KauFJfZbniCu+vrac=";
+  };
+
+  vendorHash = "sha256-UxTETtR7LNOavIS2oJRVKx/Ns2bvzJe+Enh8xFhM2mI=";
+
+  buildInputs = [
+    fuse
+  ];
+
+  checkFlags =
+    let
+      skippedTests = [
+        # mount: fusermount: exec: "fusermount": executable file not found in $PATH
+        "TestExecuteCmdMountDefault"
+      ]
+      ++ lib.optionals stdenv.isDarwin [
+        "TestBTreeScanMemory"
+        "TestBTreeScanPebble"
+      ];
+    in
+    [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
+
+  meta = {
+    mainProgram = "plakar";
+    description = "Encrypted, queryable backups for engineers based on an immutable data store and portable archives";
+    homepage = "https://www.plakar.io";
+    license = lib.licenses.isc;
+    maintainers = with lib.maintainers; [
+      heph2
+      qbit
+    ];
+  };
+})

